ui/app/pages/create-account/connect-custodial/trustvault/email/email-screen.js

```diff
@@ -0,0 +1,187 @@
+import React, { PureComponent } from 'react'
+const PropTypes = require('prop-types')
+import Button from '../../../../../components/ui/button'
+import TextField from '../../../../../components/ui/text-field'
+import { CONNECT_HARDWARE_ROUTE } from '../../../../../helpers/constants/routes'
+const ENTER_KEY = 13
+class EmailScreen extends PureComponent {
+  constructor (props) {
+    super(props)
+    this.state = {
```

Comment:
Can we move this `state` field onto the class and remove the (resulting) empty constructor?